mmdet/ops/merge_cells.py

```diff
@@ -0,0 +1,125 @@
+from abc import abstractmethod
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+
+class BaseMergeCell(nn.Module):
+
+    def __init__(self,
+                 channels=256,
+                 binary_op_type='sum',
+                 with_conv=True,
+                 conv_cfg=dict(),
+                 norm_cfg=None,
+                 order=('act', 'conv', 'norm'),
+                 with_input_conv_x=False,
+                 with_input_conv_y=False,
+                 input_conv_cfg=None,
+                 input_norm_cfg=None,
+                 resize_methods='interpolate'):
```

Comment:
- Use the singular form instead of plural form.
- `interpolate ` is inaccurate. Both `nearest` and `bilinear` are interpolations. This argument can be renamed to `interpolation='nearest'`